js/ui/pivot_grid/ui.pivot_grid.js

```diff
@@ -1177,9 +1185,9 @@ const PivotGrid = Widget.inherit({
         let rowsAreaWidth = 0;
         let hasRowsScroll;
         let hasColumnsScroll;
-        const scrollingOptions = that.option('scrolling') || {};
-        const scrollBarInfo = getScrollBarInfo(scrollingOptions.useNative);
-        const scrollBarWidth = scrollBarInfo.scrollBarWidth;
+
+        const scrollBarUseNative = this.getUseNativeOptionValue();
```

Comment:
const isNativeScrollbarInDataArea?

Reply:
I transfer this part direct to dataArea component